apps/explorer/priv/repo/migrations/20221020115129_add_block_hash_to_celo_election_rewards.exs

```diff
@@ -0,0 +1,25 @@
+defmodule Explorer.Repo.Migrations.AddBlockHashToCeloEpochRewards do
+  @doc """
```

Comment:
This should be `@moduledoc`, there's no warning because this is an `.exs` (script) file and isn't compiled